src/skottie-player.ts

```diff
@@ -104,26 +130,299 @@ export class SkottiePlayer extends LitElement {
   @property({type: Number})
   height = 0;
 
+  /**
+   * Indicator for file upload.
+   */
+  @property({type: Boolean})
+  fileUploaded = false;
+
+  /**
+   * Indicator for showing JSON editor.
+   */
+  @property({type: Boolean})
+  showJsonEditor = false;
+
+  /**
+   * The content of the uploaded file as a formatted string.
+   */
+  @property({type: String})
+  fileString = '';
+
+  /**
+   * The JSONEditor.
+   */
+  @property({type: JSONEditor})
+  jsonEditor = {};
+
+  /**
+   * The canvas surface for Skottie.
+   */
+  @property({type: Object})
+  canvasSurface = {};
+
+  /** 
+   * The user uploaded assets.
+   */
+  @property({type: Object})
+  assets = {};
+
+  /**
+   * The dominant colors of the uploaded image assets.
+   */
+  @property({type: Object})
+  imageColors = {};
+
+  /**
+   * Renders asset input buttons after json upload.
+   */
+  renderAssetInputs() {
```

Comment:
You should make sure to set the return type on functions, for example renderAssetInputs(): TemplateResult {...}

Reply:
Do this for all the other functions aswell